libraries/AP_Scripting/examples/ahrs-source-gps-optflow.lua

```diff
@@ -115,12 +112,9 @@ function update()
 
   -- get opticalflow innovations from ahrs (only x and y values are valid)
   local opticalflow_over_threshold = true
-  local opticalflow_xy_innov = 0
-  local opticalflow_innov = Vector3f()
-  local opticalflow_var = Vector3f()
-  opticalflow_innov, opticalflow_var = ahrs:get_vel_innovations_and_variances_for_source(5)
+  local opticalflow_innov = ahrs:get_vel_innovations_and_variances_for_source(5)
   if (opticalflow_innov) then
-    opticalflow_xy_innov = math.sqrt(opticalflow_innov:x() * opticalflow_innov:x() + opticalflow_innov:y() * opticalflow_innov:y())
+    local opticalflow_xy_innov = math.sqrt(opticalflow_innov:x() * opticalflow_innov:x() + opticalflow_innov:y() * opticalflow_innov:y())
```

Comment:
I guess "opticalflow_xy_innov" will exist outside the if statement?

Reply:
No, it is only used on the following line.

This is just the change that fixes the warnings of course, it does not mean that the script now works as it was originally intended.

Reply:
yup, great, thanks!